src/components/Articles.jsx

```diff
@@ -1,9 +1,35 @@
 import React from 'react'
+import { useEffect, useState } from 'react'
+import Articles_Item from './Articles_Item';
+import { getArticles } from '../utils/api';
 
-const Articles = () => {
-  return (
-    <div>Articles</div>
-  )
-}
+export default function Articles(){
+    // const [articles, setArticles] = useState("");
+    // const [articlesId, setArticlesId] = useState("");
+    const [articlesList, setArticlesList] = useState([]);
```

Comment:
Please add a state for loading and display a message to the user while the articles are loading.